src/ontology/OntoFox_outputs/OBO_imports.owl

```diff
@@ -18,6 +18,92 @@
     //
     ///////////////////////////////////////////////////////////////////////////////////////
      -->
+
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/IAO_0000111 -->
+
+    <owl:AnnotationProperty rdf:about="http://purl.obolibrary.org/obo/IAO_0000111">
+        <obo:IAO_0000111>editor preferred label</obo:IAO_0000111>
+        <obo:IAO_0000111 xml:lang="en">editor preferred label</obo:IAO_0000111>
+        <obo:IAO_0000111>editor preferred term</obo:IAO_0000111>
+        <obo:IAO_0000111 xml:lang="en">editor preferred term</obo:IAO_0000111>
+        <obo:IAO_0000111>editor preferred term~editor preferred label</obo:IAO_0000111>
+        <obo:IAO_0000111 xml:lang="en">editor preferred term~editor preferred label</obo:IAO_0000111>
+        <obo:IAO_0000111>编辑首选术语</obo:IAO_0000111>
```

Comment:
These are the sorts of annotations that we've been getting from OntoFox, but I really don't want them in OBI.

Reply:
I agree and am glad to help manually or programmatically if you want.

I agree that `src/ontology/modules/epitope-assays.owl` is missing, but I attributed that to the fact that the make processes first cat '' to the module OWL files, then try to rebuild them with `robot template`.  'Oncostatin M production' is required by  row 403 in `epitope-assays.tsv`, but it couldn't be found in `obi-edit.owl`, so that task failed. 

I haven't found anything that I can do that that point to get the module templating task to work again, because they all seem to depend on one another, in terms of being imported into `obi-edit.owl`, and therefore serving as `--input`s into each other's build.

I may have gotten some of that wrong and am not really complaining. If you think you know what I'm getting at, please set me straight when you can.

----

> echo '' > src/ontology/modules/epitope-assays.owl
java -jar build/robot.jar --prefix "REO: http://purl.obolibrary.org/obo/REO_" merge \
	--input src/ontology/obi-edit.owl \
	template \
	--template src/ontology/templates/epitope-assays.tsv \
	annotate \
	--ontology-iri "http://purl.obolibrary.org/obo/obi/dev/modules/epitope-assays.owl" \
	--output src/ontology/modules/epitope-assays.owl
MANCHESTER PARSE ERROR the expression 'has_specified_output some ('information content entity' and ('is about' some 'Oncostatin M production'))' at row 403, column 11 in table "src/ontology/templates/epitope-assays.tsv" cannot be parsed: encountered unknown 'Oncostatin M production'
For details see: http://robot.obolibrary.org/template#manchester-parse-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
make: *** [src/ontology/modules/epitope-assays.owl] Error 1
make: *** Deleting file `src/ontology/modules/epitope-assays.owl'